scripts/standby_node_health_check/standby_node_health_check.py

```diff
@@ -0,0 +1,530 @@
+                status_check = link_enabled == "1" and link_connected == "1"
+                rings_info[f"Link {current_ring} Node {node_id}"] = {
+                    "ip": current_ip,
+                    "status_text": f"node {node_id} "
+                    f"link enabled:{link_enabled} "
+                    f"link connected:{link_connected}",
+                    "status_check": status_check,
+                }
+        return rings_info
+
+    @classmethod
+    def check_if_service_is_active(cls, service_name: str):
+        """
+        service_name for example : pacemaker
+        """
+        command = cls.SYSTEMCTL_IS_ACTIVE_COMMAND.format(service_name)
+        _, stdout = cls._run_command(command)
+        if stdout == "active":
+            logger.info("Service %s is active", service_name)
+            return True
+        logger.error("Service %s is not active", service_name)
+        return False
+
+    def set_ufm_ha_cluster_status_string(self):
+        ret_code, res_string = self._run_command(self.HA_STATUS_COMMAND)
+        if ret_code != 0:
+            logger.error("Failed to get ha cluster status")
+            res_string = ""
+        self._ufm_ha_status_string = res_string
+
+    def _get_status_value_from_ha_cluster_status(self, keyword: str):
+        for line in self._ufm_ha_status_string.splitlines():
+            line = line.strip()
+            if line.startswith(keyword):
+                val = line.split(":", 1)[-1].strip().lower()
+                val = (
+                    val.replace("[", "")
+                    .replace("]", "")
+                    .replace("(", "")
+                    .replace(")", "")
+                )
+                return val
+        return None
+
+    def check_drbd_connectivity(self):
+        if not self._ufm_ha_status_string:
+            return False
+        drbd_connectivity_status = self._get_status_value_from_ha_cluster_status(
+            "DRBD_CONNECTIVITY"
+        )
+        if drbd_connectivity_status != "connected":
+            logger.error(
+                "DRBD CONNECTIVITY status is %s and not Connected",
+                drbd_connectivity_status,
+            )
+            return False
+        logger.info("DRBD CONNECTIVITY status is ok - Connected")
+        return True
+
+    def check_drdb_role(self):
+        if not self._ufm_ha_status_string:
+            return False
+        drdb_resource_status = self._get_status_value_from_ha_cluster_status(
+            "DRBD_ROLE"
+        )
+        if drdb_resource_status != "secondary":
+            logger.error(
+                "DRDB ROLE status is %s and not Secondary", drdb_resource_status
+            )
+            return False
+        logger.info("DRDB ROLE status is ok - Secondary")
+        return True
+
+    def print_summary_information(self):
+        logger.info("")
+        logger.info("Executive summary:")
+        if len(self._summary_actions) > 0:
```

Comment:
please add comment - if we have something in the summary it means we had some failures...